src/i18n/messages/es-MX.ts

```diff
@@ -64,6 +64,8 @@ export const Spanish: Messages = {
   Settings6: "Ciudades",
   Settings7: "Idioma",
   Settings8: "Próximamente Globle Edición de Ciudades",
+  Answer: "Respuesta",
+  Closest: "Los países más cercanos",
```

Comment:
Provisional translation. Please review. Not sure this is the appropriate wording in Spanish.